io/include/traccc/io/csv/particle.hpp

```diff
@@ -35,4 +38,18 @@ struct particle {
                    vt, px, py, pz, m, q);
 };
 
+inline traccc::particle make_particle_edm(
+    const traccc::io::csv::particle& csv_ptc) {
+
+    point3 pos{csv_ptc.vx, csv_ptc.vy, csv_ptc.vz};
+    vector3 mom{csv_ptc.px, csv_ptc.py, csv_ptc.pz};
+
+    traccc::particle ptc{csv_ptc.particle_id, csv_ptc.particle_type,
+                         csv_ptc.process,     pos,
+                         csv_ptc.vt,          mom,
+                         csv_ptc.m,           csv_ptc.q};
+
+    return ptc;
+}
```

Comment:
And here as well. I don't want any of these in public headers...

Reply:
Made a header and source for `make_measurement_edm`